pkgs/by-name/ll/llm-ls/package.nix

```diff
@@ -19,21 +24,43 @@ rustPlatform.buildRustPackage {
     sha256 = "sha256-ICMM2kqrHFlKt2/jmE4gum1Eb32afTJkT3IRoqcjJJ8=";
   };
 
-  cargoHash = "sha256-Fat67JxTYIkxkdwGNAyTfnuLt8ofUGVJ2609sbn1frU=";
+  cargoHash = "sha256-m/w9aJZCCh1rgnHlkGQD/pUDoWn2/WRVt5X4pFx9nC4=";
+
+  cargoPatches = [
+    # https://github.com/huggingface/llm-ls/pull/102
+    (fetchpatch {
+      name = "fix-time-compilation-failure.patch";
+      url = "https://github.com/huggingface/llm-ls/commit/cb0a3da737d933614bcd011e91048f2009036d09.patch?full_index=1";
```

Comment:
This is not a stable URL, as the commit is liable to be garbage collected if the PR is force pushed.  For unmerged PRs the patch file should be included in Nixpkgs.